roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -92,7 +92,7 @@ def load_known_checks(self):
                         check_name,
                         cls.__module__, cls.__name__,
                         other_cls.__module__, other_cls.__name__))
-            known_checks[check_name] = cls(execute_module=self._execute_module)
+            known_checks[check_name] = cls(execute_module=self._execute_module, tmp=tmp, task_vars=task_vars)
```

Comment:
This is the only place we need to pass `tmp` and `task_vars` to the check. Each check is instantiated once per task and host so the stored values are only ever used on one run.